Lib/hmac.py

```diff
@@ -142,3 +149,30 @@ def new(key, msg = None, digestmod = None):
     method.
     """
     return HMAC(key, msg, digestmod)
+
+
+def digest(key, msg, digest):
+    """Fast inline implementation of HMAC
```

Comment:
describe the parameters so that interactive users doing help(hmac.digest) see something useful.